src/tests/ftest/launch.py

```diff
@@ -1292,7 +1298,16 @@ def install_debuginfos():
     cmds.append(cmd)
 
     for cmd in cmds:
-        print(get_output(cmd))
+        try:
+            print(get_output(cmd, raise_exception=True))
+        except RuntimeError as error:
+            print(error)
+            print("Going to refresh caches and try again")
+            cmd_prefix = ["sudo", "yum", "--enablerepo=*debug*"]
+            cmds.insert(0, cmd_prefix + [ "clean", "all"])
```

Comment:
If you hit multiple errors don't you only want to run "sudo yum --enablerepo=*debug* clean all" once?  should this be more like:
```
    yum_clean_cmd = ""
    for cmd in cmds:
        try:
            print(get_output(cmd, raise_exception=True))
        except RuntimeError as error:
            print(error)
            if not yum_clean_cmd:
                print("Going to refresh caches and try again")
                yum_clean_cmd = [
                    "sudo", "yum", "--enablerepo=*debug*", "clean", "all"]
    if yum_clean_cmd:
        cmds.insert(0, yum_clean_cmd)
```

Reply:
Oh right.  I think my original thought was to just abort the processing of `cmds`, run the _clean_ and then try them all again since they are idempotent.  Let me look into that...